cluster.go

```diff
@@ -56,6 +56,22 @@ func (c *ClusterClient) Watch(keys ...string) (*Multi, error) {
 	return client.Watch(keys...)
 }
 
+// PoolMetrics returns accumulated connection pool metrics
+func (c *ClusterClient) PoolMetrics() *PoolMetrics {
```

Comment:
Probably return map[string]*PoolStats?

Or it is good time to add iterator like you suggested? E.g.

```
clients := cluster.Masters()
for _, client := range clients {
    fmt.Println(client.PoolStats())
}
```